joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/passes/TypeRecoveryPassTests.scala

```diff
@@ -214,6 +215,23 @@ class TypeRecoveryPassTests extends PySrc2CpgFixture(withOssDataflow = false) {
       "bar.py"
     ).cpg
 
+    "be able to traverse from `foo.[x|y|db]` to its members" in {
```

Comment:
One additional (non-data-flow) test to show recovered member references that were previously not there